E2E-tests/config/substrate/staging_stack.json

```diff
@@ -4,7 +4,7 @@
         "ogmios_port": 1337,
         "kupo_host": "staging-preview-services-service.staging-preview.svc.cluster.local",
```

Comment:
Irrelevant with this PR, just thought we should change the name of the file to staging_preview_stack.json

Reply:
staging -> staging_preview